kwanyong/.env.sample

```
@@ -0,0 +1,11 @@
+DB_CONNECTION = mysql
+DB_HOST = 127.0.0.1
+DB_USERNAME = username
+DB_PASSWORD = password
+DB_DATABASE = westagram_46
+DB_PORT = 3306
+DB_LOGGING =TRUE
+
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/westagram_46"
+
+PORT = 8000
```

sample 파일에서는 실제 서비스에서 사용하는 환경변수에 대한 간략한 설명을 작성해주시면 됩니다!
실제 값들 혹은 실제 값을 유추할 수 있는 값들이 적혀있는데 아래와 같이 수정해보면 어떨까요?
또한, 변수와 값 사이의 공백은 지우는 것을 권장드립니다!
```suggestion
DB_CONNECTION=mysql
DB_HOST=host
DB_USERNAME=username
DB_PASSWORD=password
DB_DATABASE=db_name
DB_PORT=port
DB_LOGGING=boolean

DATABASE_URL="mysql://username:password@host:port/db_name"

PORT=port
```